src/components/Toast/CloseIconButton/CloseIconButton.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { IconButton } from '../../IconButton';
+import { ReactComponent as PlusIcon } from '../../../assets/icons/plus.svg';
```

Comment:
Please use this icon https://zeroheight.com/7sjjzhgo2/p/884c1f/i/3206562 instead of the Plus.